source/includes/api/_widget.md

```diff
@@ -1,428 +1,296 @@
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?verified=false
+curl -X GET "https://api.resourcewatch.org/v1/widget?default=false"
 ```
 
-> Return the widgets filtered by template
+> Filter widgets by environment
+
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?template=true
+curl -X GET "https://api.resourcewatch.org/v1/widget?env=staging"
 ```
 
-> Filter widgets by default option
+> Return the widgets filtered by those whose applications contain both `rw` and `prep` applications simultaneously
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?default=true
+curl -X GET "https://api.resourcewatch.org/v1/widget?app=rw@prep"
 ```
 
-> Return widgets whose applications contain rw
+The widget list endpoints provide a wide range of filters that you can use to tailor your widget listing. The great majority of the widget fields you'll find on the [widget reference](#widget-reference) section can be used as filters when listing widgets, with the following exceptions:
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?app=rw
-```
+- `id`
+- `userName`
+- `userRole`: filtering by the role of the owning user can be done using the `user.role` query argument.
 
-### Pagination params
+Additionally, you can use the following filters:
 
-Field        |       Description        |   Type
------------- | :----------------------: | -----:
-page[size]   | Number elements per page. There's a limit of 100 to the size of each page, which is not being enforced at the moment, but queries for larger page sizes are not supported. This means future requests may fail if not respecting the page size limit. | Number
-page[number] |      Number of page      | Number
+- `collection`: filters by a [collection](#collections) id. Requires being authenticated.
+- `favourite`: if any value is defined for this query param, only the widgets set by the user as [favorites](#favorites) will be returned. Requires being authenticated.
 
-> Return the widgets from page 2, with 5 elements per page
+Multiple parameters can be combined into a complex `and` logic filter. Depending on the type of the field you're filtering by, the following behavior is available:
 
-```shell
-curl -X GET https://api.resourcewatch.org/v1/widget?page[size]=5&page[number]=2
-```
+- String type fields support both regular expressions and partial matches.
+- Array type fields support `,` as `OR` and `@` as `AND` separators for multiple values.
+- Object type fields will match the whole object, not just its parts.
 
 ### Sorting
 
-The API currently supports sorting by means of the `sort` parameter. Sorting can be done using any field from the widget, as well as `user.name` and `user.role` (sorting by user data is restricted to ADMIN users).
+The API currently supports sorting by means of the `sort` parameter. Sorting can be done using any field from the widget, as well as `user.name` and `user.role` (sorting by user data is restricted to `ADMIN` users).
```

Comment:
Same question as for filtering - can we separate this out into its own more general sorting section?

Reply:
seems like this sorting logic is used on datasets, layers, widgets and metadata. I suspect it's not used anywhere else - I checked vocabulary, area, geostore and subscriptions, and none of them have it. 

I honestly have no good reason to pick one over the other - centralizing is better if things are consistent (I need to double-check this) but it means more work if things diverge. Engineer laziness has a slight preference towards keeping things as is, but I'll leave it up to you to decide

Reply:
Yeah I guess it depends on consistency - is the sorting/filtering logic implemented separately for all of these resources, or is it a common layer on top?

If it's consistent, I would add a "Sorting & Filtering" section at the top and then have a stub in each of the resources that supports it that says please see the sorting and filtering section above.

Reply:
No, each MS has its own specific code for sorting, it just happens to be copy-pasted from one to another - to the point where there's a bug in that code, that now only exists in metadata because I fixed it for dataset widget and layer as I wrote these docs.

Reply:
Yeah let's leave it...potentially could consider fixing these bugs and implementing generic sorting and filtering middleware for the next contract